src/decorators/nanp.ts

```diff
@@ -37,6 +41,11 @@ const decorator: IDecorator = {
 
       if (i >= 0 && i <= 2) {
         decoratedNumber.parts.push(areaCodePart(digit));
+      } else if (digit === "x") {

```

Comment:
Again, this isn't really a known convention, and I'm not even sure it's a sensible default outside of English language countries. I think passing this in on instantiation as well is most sensible.